src/arrayMethodJoin.js

```diff
@@ -5,7 +5,20 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
Separator could be any, not only comma, comma should be by dafult
```suggestion
  [].__proto__.join2 = function(separator = ',') {
```